src/components/Issue/CreateEditIssue/CreateEditIssue.tsx

```diff
@@ -208,9 +209,9 @@ class CreateEditIssue extends Component<
             variant="outlined"
             value={status}
           >
-            {Object.values(Status).map((status, index) => (
-              <MenuItem key={index} value={status}>
-                {status}
+            {this.props.statuses.map((status, index) => (
```

Comment:
In this scope, I think there are multiple variables with the same name - status (there is shadowing). Not sure but eslint should give a warning about this.

Reply:
done